megaavr/extras/Ref_TCD.md

```diff
@@ -16,6 +16,10 @@ analogWrite(PIN_PA6, 64); // 25% PA4, 25% PA6, 75% PA7 - PA4 and PA6 are both ch
 // or call turnOffPwm(pin);
 ```
 
+## WOC and WOD (PC0 and PC1)
+
+On 20/24 pin 1-Series parts, `TCD0` can also drive the `WOC` and `WOD` pins (PC0 and PC1 respectively). `WOC` and `WOD` can each be connected to either `WOA` or `WOB`, but the core only supports connecting `WOC` to `WOA` and `WOD` to `WOB`.
```

Comment:
The changes in this document do not represent the state of the functionality presently available in megaTinyCore. 

We actually use WOC and WOD by default, because we run TCA0 in splitmode (analogWrite is 8-bit) so the WOA and WOB don't become intereresting until pins PC4 and PC5 become available (24 pinonly), at which point you can switch WO5 and WO4 to those pins, and use PA4/PA5 for TCD. In github version the tools PWM pins menu should work for setting the pins to get PWM out of. (there were some bugs in current release)